user/models.py

```diff
@@ -1,5 +1,49 @@
-from django.contrib.auth.models import AbstractUser
+from django.contrib.auth.base_user import BaseUserManager
+from django.contrib.auth.models import (
+    AbstractUser,
+    UserManager as DgangoUserManager,
```

Comment:
There is a typo in the import statement: `DgangoUserManager` should be `DjangoUserManager`. Please correct this to avoid any potential issues with the user manager functionality.